### PR TITLE
[landing page] career banner

### PR DESCRIPTION
**Pull request by palmfjord**

### CODE REVIEW

File: src/sections/landing/career-banner.tsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+  }),
+);
+class CareerBanner extends React.PureComponent<Props, { hasMounted: boolean }> {
+  state: { hasMounted: boolean } = { hasMounted: false };
+  render() {
+    return (
+      <CareerBannerContainer className="Container">
+        <VisibilitySensor
+          onChange={(isVisible: boolean) => {
+            this.setState(({ hasMounted }) => ({
```
Comment (Iamsamwhoami) [inline src/sections/landing/career-banner.tsx]: Can we use `constate` for this?

File: src/sections/landing/career-banner.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        textDecoration: 'none',
+      },
+
+      opacity: 0,
+      animation: isVisible ? `${fadeIn} 400ms forwards` : 'none',
+      animationDelay: `${animationDelay}ms`,
+    },
+  }),
+);
+class CareerBanner extends React.PureComponent<Props, { hasMounted: boolean }> {
```
Comment (Iamsamwhoami) [inline src/sections/landing/career-banner.tsx]: Can we not use classes? 😄 (I dislike them strongly 🙈)

File: src/sections/landing/career-banner.tsx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+            this.setState(({ hasMounted }) => ({
+              hasMounted: hasMounted || isVisible,
+            }));
+          }}
+        >
+          {() => (
+            <>
+              <HedvigLogo
+                animationDelay={0}
+                src="/assets/identity/hedvig-symbol-color.svg"
```
Comment (palmfjord) [inline src/sections/landing/career-banner.tsx]: lets inline this svg instead

File: src/templates/landing-page.js [hunk 4]
```diff
@@ -402,9 +393,9 @@ class LandingTemplate extends React.Component {
               </div>
             </div>
 
-            {/* Media logos on mobile */}
+            {/* Media logos on desktop */}
```
Comment (palmfjord) [inline src/templates/landing-page.js]: just `media logos`

File: src/sections/landing/career-banner.tsx [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+  }),
+);
+
+const actions = {
+  mount: () => (_: State): Partial<State> => ({ hasMounted: true }),
+};
+
+const CareerBanner: React.SFC<Props> = (props) => (
+  <CareerBannerContainer className="Container">
+    <Container<State, Actions>
```
[the remark this answers is not in the document]
Comment (palmfjord) [reply src/sections/landing/career-banner.tsx]: approval plz? ✅🙏

File: src/sections/landing/career-banner.tsx [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+      marginLeft: 'auto',
+      background: colors.PURPLE,
+      padding: '10px 25px',
+      fontSize: 16,
+      color: '#fff',
+      textDecoration: 'none',
+      borderRadius: '50px',
+      transition: 'background 400ms',
+      '&:hover, &:focus': {
+        background: '#752EFF',
```
Comment (hedvigoscar) [inline src/sections/landing/career-banner.tsx]: What color is this?

Comment (palmfjord) [reply src/sections/landing/career-banner.tsx]: hehe, nice catch, tots forgot about this. its basically lightened `colors.PURLE`. What do we usually do in this case?

File: src/sections/landing/career-banner.tsx [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+    animationDelay,
+    isVisible,
+  }: { animationDelay: number } & WithVisibility) => ({
+    '&&': {
+      display: 'inline-block',
+      marginLeft: 'auto',
+      background: colors.PURPLE,
+      padding: '10px 25px',
+      fontSize: 16,
+      color: '#fff',
```
[the remark this answers is not in the document]
Comment (palmfjord) [reply src/sections/landing/career-banner.tsx]: sure!

**Pull request merged**